cmd/validator/validator.go

```diff
@@ -235,7 +325,13 @@ func mainInit() int {
 	// since the exclude dirs are a comma separated string
 	// it needs to be split into a slice of strings
 	excludeDirs := strings.Split(*validatorConfig.excludeDirs, ",")
-	choosenReporter := getReporter(validatorConfig.reportType, validatorConfig.output)
+
+	chosenReporters := make([]reporter.Reporter, 0)
+	for reportType, outputFile := range validatorConfig.reportType {
+		rt, of := reportType, outputFile // avoid "Implicit memory aliasing in for loop"
```

Comment:
This could be removed after go 1.22

I didn't check what version is currently supported

Reply:
Yes this can be removed if upgraded to 1.22

Reply:
Let's do 1.22 upgrade in a separate story. I think this is fine for now